src/load/azext_load/tests/latest/test_load_test.py

```diff
@@ -148,17 +148,29 @@ def test_load_test_create(self, rg, load, vnet):
         pass_fail_metric = response.get("passFailCriteria", {}).get(
             "passFailMetrics", {}
         )
+        # Do not perform equality checks with floating point values.
+        # Use a tolerance value to check if the values are close enough.
         for item in pass_fail_metric.values():
             if item.get("clientMetric") == "requests_per_sec":
-                assert item.get("value") == 78.0
+                assert abs(item.get("value") - 78.0) < 1e-6
                 assert item.get("condition") == ">"
                 assert item.get("aggregate") == "avg"
             elif item.get("clientMetric") == "error":
-                assert item.get("value") == 50.0
+                assert abs(item.get("value") - 50.0) < 1e-6
```

Comment:
move to a const and use it in all these tests ?

Reply:
updated